src/Homepage/ResultBlock.js

```diff
@@ -81,6 +82,11 @@ function ResultBlock(props) {
               textToHighlight={result.name}
             />
           </h5>
+          <div className="mt-n1">
+            <small className="text-muted d-inline-block mt-2">
+              <strong>Within {distanceKM} km</strong>
```

Comment:
Why do we need that extra wrapping div with negative margin 🤔
This markup generates the same layout

```suggestion
            </h5>
            <small className="text-muted d-block mt-1">
              <strong>Within {distanceKM} km</strong>
            </small>  
```

Reply:
Good catch, I changed it to `mt-1,` but it was originally in `<div>` to be on a separate block.